### PR TITLE
Allow to easily publish wheels

### PR DESCRIPTION
**Pull request by xavfernandez**
`python setup.py sdist bdist_wheel upload` will then publish wheel and sdist to PyPI.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```
Comment (ericpruitt) [inline setup.cfg]: Could you please add a space between the equal sign here on both sides? In the setup.py file, I more or less follow PEP 8, so spaces are added around "=" when it is not an argument keyword assignment?

Comment (xavfernandez) [inline setup.cfg]: done :)

**Comment by ericpruitt** [comment]
I have merged this change as 2630ed29e447da216a6b0119de6eef2f4212506f, and I also made a small change in f287c30d27d6ca65df3abdfa57265e690396f7ad so the setup.py script will work with both distutils and setuptools. I am going to close this for now, but please let me know if you have any problems with the change and re-open this if you feel it's necessary. Thank you for the pull request.

**Comment by xavfernandez** [comment]
Thanks for merging and thanks a lot for cronex :-)

**Pull request closed without merging**